app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -10,6 +10,7 @@ import io.reactivex.rxkotlin.zipWith
 import org.simple.clinic.ReplayUntilScreenIsDestroyed
```

Comment:
There is an extra "adds" in the commit message.

Reply:
Removed.